### ISSUE TITLE
How to bind a key that emits multiple keycodes?

### ISSUE THREAD

**Issue by pimartinet** [kmonad/kmonad#487]
Hello,
I have a microsoft keyboard with an ‘office’ key which emits `Ctrl_L + Shift_L + Alt_L + Super_L` and an ‘emoji’ key that emits `Ctrl_L + Shift_L + Alt_L + Super_L + space`.
Is there (or can there) be a way to bind those keys in KMonad?

**Comment by slotThe** [comment]
Are you looking for things like binding a key to `C-S-A-M-spc`?  You can just create a `defalias` with exactly that sequence

**Comment by pimartinet** [comment]
Yes, that's what I'd like to do. But I would need to use the `defalias` definition in the `defsrc` block, which is as I understand it not allowed.

**Comment by lenalebt** [comment]
Did you find a solution? I think I have the same problem. I have a bluetooth numpad keyboard that I would like to use as a specific macro keyboard. That 28-keys keyboard has a few keys that I could not map so far: `%`, `(` and `)` are present, which send `SHIFT+5`, `SHIFT+9` and `SHIFT-0`. I was able to remap all other keys, but those three are missing. Is there a way to grab them "all"?

I worked around by capturing `5`, `9` and `0`, but this will then emit `SHIFT` together with my remapped keys.
